tunnel-server/deployment/k8s/README.md

```diff
@@ -0,0 +1,100 @@
+# Deploying your own instance of the Tunnel Server
+
+This directory contains an example deployment of the Tunnel Server on Kubernetes.
+
+Note that this is an advanced task which requires some networking and Kubernetes know-how.
+
+## Why
+
+Deploying a private instance of the Tunnel Server allows for fine-grained control of:
+
+- The URLs created for preview environments: e.g, use a custom domain.
+- Geolocation of the server: reduced distance to environments can result better network performance.
+- Security and privacy: deploy everything in your VPC, no traffic to 3rd parties.
+
+## Requirements
+
+- A Kubernetes cluster
+- An ingress solution to make K8S Services accesible from your network (e.g, Traefik). In this example, we'll use your cloud provider's load balancer.
```

Comment:
The "An ingress solution" might be confusing here due to the ambiguity with k8s ingress resources. 
Since Traefik and cloud provider load balancer (based on k8s service-> loadbalancer) work at different layers (l4 vs l7), and the sentence imply that we're using the cloud provider's load balancer instead (which is incorrect, since we'll need service->loadbalancer endpoint to expose traefik if we were to use it).
In addition, cloud providers has both solutions for l4 and l7 integrated with Kubernetes (via k8s service->loadbalancer and k8s gateway/ingress) so it's not clear which one are we referring to.